tests/unit/local/lambdafn/test_runtime.py

```diff
@@ -81,6 +81,7 @@ def test_must_create_lambda_container(self, LambdaContainerMock):
             debug_options=debug_options,
             env_vars=self.env_var_value,
             memory_mb=self.DEFAULT_MEMORY,
+            container_host=None,
```

Comment:
Isn't `container_host` already `None` by default?

Reply:
Yes lambda container is called with `container_host = None` by default. However if we don't set the None here, the call assertion will fail.